winit/src/system.rs

```diff
@@ -0,0 +1,53 @@
+//! Access the native system.
+use crate::command::{self, Command};
+pub use iced_native::system::*;
+
+use iced_graphics::window;
+
+/// Query for available system information.
+///
+/// Returns `None` if not using the `sysinfo` feature flag.
+pub fn information<Message>(
+    f: impl Fn(Option<Information>) -> Message + 'static,
+) -> Command<Message> {
+    Command::single(command::Action::System(Action::QueryInformation(
+        Box::new(f),
+    )))
+}
+
+#[cfg(feature = "sysinfo")]
+pub(crate) fn get_information(
+    graphics_info: &window::GraphicsInformation,
+) -> Option<Information> {
+    use sysinfo::{ProcessExt, ProcessorExt, System, SystemExt};
+    let mut system = System::new_all();
```

Comment:
Because of https://github.com/iced-rs/iced/commit/5eefa5d4ead9ebfac7dab1db9aebf9797d2dad38, I've decided to maintain this here (this should isolate the crate to the Action itself).
And it seems that refreshing and creating are almost the same.